pyproject.toml

```diff
@@ -55,6 +55,7 @@ dependencies = [
   "databricks-labs-pylint~=0.4.0",
   "mypy~=1.10.0",
   "numpy==1.26.4",
+  "cryptography>=41.0.3"
```

Comment:
this is test-time dependencies and it's installed only on CI and developer machines, you probably intend this library to be executed on customer machines - move it to line 19. also what license is this library? does it require any compiled dependencies? we also need to reflect it in `NOTICE` file.

Reply:
Thanks, corrected the code.
It is apache2.0 license https://github.com/pyca/cryptography?tab=Apache-2.0-2-ov-file
and According to the internet, it requires the below-complied dependencies.

```
C Compiler: A tool that translates C code into machine code. Examples include gcc (GNU Compiler Collection) or clang.
OpenSSL: A software library that provides cryptographic functions. It must be installed on your system because the cryptography library relies on it for certain operations.
```